### PR TITLE
Update demo

### PR DESCRIPTION
**Pull request by LukasVeezoo**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -14,29 +14,28 @@ def generate_jwt(payload, private_key, expiration_minutes=30):
         "typ": "JWT",
         "alg": "RS256"
     }
-    # "iat" (Issued At) claim
```
Comment (tillhaug) [inline app.py:17]: any particular reason why we drop the comments?

File: app.py [hunk 2]
```diff
@@ -90,7 +89,7 @@ def Oliver():
     jti = str(uuid.uuid1())
     t_payload = {
         # This is the ID for the tenant you configured in Veezoo Admin in a previous step
-        "tenantId": "2",
+        "tenantId": "Office Supplies",
```
Comment (tillhaug) [inline app.py]: this doesn't look like a "typical id", not that it doesn't work, was just a bit surprised, maybe could do office_supplies instead?

**Comment by tillhaug** [comment]
hey @LukasVeezoo I have added two comments. Also the commit texts can be more descriptive. Why do we need to update it in the first place (?)

File: app.py (hunk 1 above)
Comment (LukasVeezoo) [reply app.py:17]: I wanted to clean it up a bit, some comments didn't seem too helpful to me

File: app.py (hunk 2 above)
Comment (LukasVeezoo) [reply app.py]: Changed in the next commit

**Pull request merged**